apps/types_/kubernetes_deployment_manifest.py

```diff
@@ -1,19 +1,39 @@
+import os
 import subprocess
 import uuid
 from datetime import datetime
+from pathlib import Path
+from typing import NamedTuple
 
 import kubernetes_validate
 import yaml
 import yaml.scanner
+from django.conf import settings
+
+from studio.utils import get_logger
+
+logger = get_logger(__name__)
+
+
+class K8SDeploymentFiles(NamedTuple):
+    values_file: Path
+    deployment_file: Path
+
+
+class ValidationResult(NamedTuple):
```

Comment:
Maybe like this?:)

```suggestion
class K8SManifestValidationResult(NamedTuple):
```

Or if you'd like to save characters:
```suggestion
class K8SManifestValidRes(NamedTuple):
```